intel-vtune.spec

```diff
@@ -0,0 +1,25 @@
+### RPM external intel-vtune 2017.0.2.478468
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/intel-vtune.xml
+<tool name="intel-vtune" version="%{realversion}">
+  <info url="https://software.intel.com/en-us/intel-vtune-amplifier-xe"/>
+  <lib name="ittnotify"/>
+  <client>
+    <environment name="INTEL_VTUNE_BASE" default="/afs/cern.ch/sw/IntelSoftware/linux/x86_64/xe2017/vtune_amplifier_xe_%{realversion}"/>
+    <environment name="INCLUDE" default="$INTEL_VTUNE_BASE/include"/>
+    <environment name="LIBDIR" default="$INTEL_VTUNE_BASE/lib64"/>
+    <environment name="BINDIR" default="$INTEL_VTUNE_BASE/bin64"/>
+  </client>
+  <runtime name="PATH" value="$INTEL_VTUNE_BASE/bin64" type="path"/>
+  <runtime name="VTUNE_AMPLIFIER_XE_2017_DIR" value="$INTEL_VTUNE_BASE"/>
+  <runtime name="INTEL_LICENSE_FILE" value="28518@lxlicen01.cern.ch,28518@lxlicen02.cern.ch,28518@lxlicen03.cern.ch" type="path" handler="warn"/>
+</tool>
+EOF_TOOLFILE
+
+## IMPORT scram-tools-post
```

Comment:
scramv1-tool-conf.file (which collects all toolfiles and bundle them in tool-conf) needs to access the ```<tool>_ROOT``` env. So each tool should set it. 
If you provide an empty %post section then cmsBuild is anyway going to generate init.sh with default values for ROOT, VERSION, PATH, LD_LIBRARY_PATH etc (just a standard init.sh file). As for toofiles we do not need any of these what is why we re-create init.* with only _ROOT variable in them.